Sources/Sentry/SentrySDK.m

```diff
@@ -149,6 +149,7 @@ + (void)startWithOptionsObject:(SentryOptions *)options
 
     SentryClient *newClient = [[SentryClient alloc] initWithOptions:options];
     [newClient.fileManager moveAppStateToPreviousAppState];
+    [newClient.fileManager moveBreadcrumbsToPreviousBreadcrumbs];
```

Comment:
`m`: This new functionality misses tests.

Reply:
Actually it's used by `testReadPreviousBreadcrumbs` so it is covered.

Reply:
Ah wait I'm wrong, this is about SentrySDK itself. Yes, will add.

Reply:
Done